update_cli_docs.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+set -euo pipefail
+
+echo "Cardano SL CLI docs updating"
+
+readonly CARDANO_DOCS_REPO="${HOME}"/cardanodocs
+readonly DOCUMENTER=cardano-cli-docs
+readonly RAW_CHAPTER_NAME="${DOCUMENTER}".md
+readonly PATH_TO_REAL_CHAPTER="${CARDANO_DOCS_REPO}"/_docs/technical/2017-01-15-cli-options.md
+readonly PATH_TO_DIR_WITH_EXECUTABLES=$(stack path --local-install-root)/bin
+
+echo "**** 1. Get raw Markdown chapter ****"
+stack exec --nix -- "${DOCUMENTER}" --bin-dir "${PATH_TO_DIR_WITH_EXECUTABLES}"
```

Comment:
We don't force `--nix` anywhere. It ought to be set in `~/.stack/config.yaml` or something.